js/utils/mathutils.js

```diff
@@ -47,29 +47,31 @@ class MathUtility {
             return Math.floor(Math.random() * (Number(n2) - Number(n1) + 1) + Number(n1));
         };
 
-        let GetRandomSolfege = (n1, n2, octave) => {
-            // [n1, n2] = n1 > n2 ? [n2, n1] : [n1, n2];
-            let lowerOctave = n1 > n2 ? -1 : 0;
-
-            let res;
-            if (n1 > n2) {
-                let choice = Math.random();
-                if (choice >= 0.5) {
-                    res = Math.floor(Math.random() * (Number(6) - Number(n1) + 1) + Number(n1));
-                    console.log(res);
-                } else {
-                    res = Math.floor(Math.random() * (Number(n2) + 1));
-                    console.log(n2, res);   
-                }
-            } else {
-                res = Math.floor(Math.random() * (Number(n2) - Number(n1) + 1) + Number(n1));
+        let GetRandomSolfege = (a1, a2, octave) => {
+            if (octave === undefined) {
+                octave = 4;
             }
             
-            if (n1 <= res && res <= 6) {
-                octave += lowerOctave;
+            let broadScale = [];
+            for(let i = octave; i <= octave+1; i++) {
```

Comment:
You could simplify this further by using .concat() to merge two copies of the array.